### PR TITLE
VZ-6666 Uptake Rancher version 2.6.7

### PR DESCRIPTION
**Pull request by pshirodkar-orcl**
VZ-6666 Uptake Rancher version 2.6.7

### CODE REVIEW

**Comment by pshirodkar-orcl** [comment]
I intentionally did not cherry-pick commits from these PRs since I cannot just cherry-pick them as-is since they contain go.mod etc changes which are dependent on security reports.
https://github.com/verrazzano/rancher/pull/42
https://github.com/verrazzano/rancher/pull/44
https://github.com/verrazzano/rancher/pull/45

File: Dockerfile.dapper [hunk 1]
```diff
@@ -97,16 +105,12 @@ RUN ln -s /usr/bin/cni /usr/bin/bridge && \
     ln -s /usr/bin/xtables-legacy-multi /usr/bin/ip6tables-save && \
     ln -s /usr/bin/xtables-legacy-multi /usr/bin/ip6tables-restore && \
     ln -s /usr/bin/xtables-legacy-multi /usr/bin/ip6tables-translate && \
-    ln -s /etc/rancher/k3s/k3s.yaml /go/src/github.com/rancher/rancher/.kube/k3s.yaml
-
+    ln -s /etc/rancher/k3s/k3s.yaml /go/src/github.com/rancher/rancher/.kube/k3s.yaml;
```
Comment (fredtibbitts) [inline Dockerfile.dapper:108]: I don't think it should matter, but I'm curious, why did you add a semicolon to this line?

File: cmd/agent/main.go [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// Copyright (C) 2020, 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline cmd/agent/main.go:1]: 2021 should be 2022

File: package/Dockerfile.agent [hunk 3]
```diff
@@ -1,16 +1,33 @@
+# Copyright (C) 2020, 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline package/Dockerfile.agent:1]: 2021 should be 2022

File: package/entrypoint.sh [hunk 4]
```diff
@@ -1,11 +1,16 @@
 #!/bin/bash
-
+# Copyright (C) 2020, 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline package/entrypoint.sh:2]: 2021 should be 2022

File: package/jailer.sh [hunk 5]
```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+# Copyright (C) 2020, 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline package/jailer.sh:2]: 2021 should be 2022

File: pkg/image/export/main.go [hunk 6]
```diff
@@ -1,3 +1,6 @@
+// Copyright (C) 2020, 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline pkg/image/export/main.go:1]: 2021 should be 2022

File: pkg/image/resolve.go [hunk 7]
```diff
@@ -1,3 +1,6 @@
+// Copyright (C) 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline pkg/image/resolve.go:1]: 2021, 2022

File: scripts/package [hunk 8]
```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
+# Copyright (C) 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline scripts/package:2]: 2021, 2022

**Review by fredtibbitts — COMMENTED** [review]
Several of the files don't have updated copyright years. If you just pulled those changes from the 2.6.6 release then we likely missed updating the copyrights in both the 2.6.6 and 2.6.5 branches.

Regarding the THIRD_PARTY_LICENSES.txt file, that needs to be copied exactly from the BA, so I just want to confirm that they match exactly.

File: Dockerfile.dapper (hunk 1 above)
Comment (pshirodkar-orcl) [reply Dockerfile.dapper:108]: Previous release https://github.com/verrazzano/rancher/blob/oracle/release/2.6.6/Dockerfile.dapper#L106 had the semicolon. So I added it. The new upstream https://github.com/rancher/rancher/blob/v2.6.7/Dockerfile.dapper#L100 does not have it - let me know if you want me to remove it.

File: cmd/agent/main.go (hunk 2 above)
Comment (pshirodkar-orcl) [reply cmd/agent/main.go:1]: Copyrights were not updated in the `oracle/release/2.6.6` branch. Let me fix in my PR.

Comment (pshirodkar-orcl) [reply cmd/agent/main.go]: Fixed

File: package/Dockerfile.agent (hunk 3 above)
Comment (pshirodkar-orcl) [reply package/Dockerfile.agent]: Fixed

File: package/entrypoint.sh (hunk 4 above)
Comment (pshirodkar-orcl) [reply package/entrypoint.sh]: Fixed

File: package/jailer.sh (hunk 5 above)
Comment (pshirodkar-orcl) [reply package/jailer.sh]: Fixed

File: pkg/image/export/main.go (hunk 6 above)
Comment (pshirodkar-orcl) [reply pkg/image/export/main.go]: Fixed

File: pkg/image/resolve.go (hunk 7 above)
Comment (pshirodkar-orcl) [reply pkg/image/resolve.go]: Fixed

File: scripts/package (hunk 8 above)
Comment (pshirodkar-orcl) [reply scripts/package]: Fixed

File: Dockerfile.dapper (hunk 1 above)
Comment (fredtibbitts) [reply Dockerfile.dapper:108]: I think it would be good to remove it, so that it's one less diff with the upstream.

**Review by fredtibbitts — APPROVED** [review]
